docs/guide/db-migrations.md

```diff
@@ -788,6 +788,13 @@ yii migrate/redo 3      # redo the last 3 applied migrations
 
 > Note: If a migration is not reversible, you will not be able to redo it.
 
+## Refreshing Migrations <span id="refreshing-migrations"></span>
+
+Since Yii 2.0.13 you can reset the whole database and apply all migrations from the beginning.
```

Comment:
That statement may need a change.
This is not entirely true, only the tables will be erased and the database has other elements like functions and stored procedures.
You only guarantee a "complete reset" using "DROP/CREATE DATABASE".
cc: @samdark

Reply:
https://github.com/yiisoft/yii2/commit/a3cd99d77d9748d4edafd28ddecd7d32d7e1a3ee